Lombiq.Hosting.Tenants.IdleTenantManagement.Tests.UI/Extensions/TestCaseUITestContextExtensions.cs

```diff
@@ -9,18 +9,22 @@ namespace Lombiq.Hosting.Tenants.IdleTenantManagement.Tests.UI.Extensions;
 public static class TestCaseUITestContextExtensions
 {
     public static async Task TestIdleTenantManagerBehaviorAsync(
-        this UITestContext context)
+        this UITestContext context,
+        string recipeId = null)
     {
         // Setting up new tenant to test the feature
         await context.CreateAndSwitchToTenantManuallyAsync(IdleTenantName, IdleTenantPrefix, string.Empty);
 
+        // This is needed for testing in NuGet as the RecipeId is different.
```

Comment:
Comment not needed, since this shouldn't had have a hard-coded setup recipe name here anyway.

Reply:
I don't even know why this wasn't failing up until now. The Nuget uses a different named recipe so it should have failed since the begging.